.github/workflows/release-workflow.yml

```diff
@@ -112,12 +113,17 @@ jobs:
 
       - name: Set next SDK version
         run: |
-          git checkout master
+          git checkout "release/${{ github.event.inputs.current_version }}"
           sed -i -r "s#version = ([^\']+)#version = ${{ github.event.inputs.next_version }}-SNAPSHOT#" gradle.properties
           git add gradle.properties
           git commit -m "CI/CD: set next version: ${{ github.event.inputs.next_version }}"
           git push
 
+      - name: Create release pull request
+        run: gh pr create -B master -H "release/${{ github.event.inputs.current_version }}" --title 'Merge release/${{ github.event.inputs.current_version }} into master' --body 'Created by Github action'
+        env:
+          GITHUB_TOKEN: ${{ secrets.CD_GITHUB_TOKEN }}
+
       - name: Checkout Swazzler
```

Comment:
I think I still need to change this PR, to create a release branch in the swazzler.